infrastructure/src/main/resources/setup/database/schema.sql

```diff
@@ -0,0 +1,284 @@
+CREATE TABLE AccessTag (
+    id            BIGINT AUTO_INCREMENT PRIMARY KEY,
+    description   VARCHAR(255) NULL,
+    name          VARCHAR(255) NULL,
+    owner_id      BIGINT       NULL,
+    parent_tag_id BIGINT       NULL,
+    CONSTRAINT access_tag_name_idx UNIQUE (name),
+    CONSTRAINT FK5q8ww38t2jsgfy73t0e12am52 FOREIGN KEY (parent_tag_id) REFERENCES AccessTag (id)
+);
```

Comment:
I think `owner_id` and `parent_tag_id` are not required

Reply:
👍